specification/monitor/resource-manager/Microsoft.Insights/stable/2023-10-01/metricDefinitions_API.json

```diff
@@ -0,0 +1,384 @@
+            "schema": {
+              "$ref": "#/definitions/MetricDefinitionCollection"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "x-ms-examples": {
+          "Get Metric Definitions without filter": {
+            "$ref": "./examples/GetMetricDefinitions.json"
+          },
+          "Get Application Insights Metric Definitions without filter": {
+            "$ref": "./examples/GetMetricDefinitionsApplicationInsights.json"
+          },
+          "Get StorageCache Metric Definitions with metric class": {
+            "$ref": "./examples/GetMetricDefinitionsMetricClass.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "MetricAvailability": {
+      "type": "object",
+      "properties": {
+        "timeGrain": {
+          "type": "string",
+          "format": "duration",
+          "description": "the time grain specifies the aggregation interval for the metric. Expressed as a duration 'PT1M', 'P1D', etc."
+        },
+        "retention": {
+          "type": "string",
+          "format": "duration",
+          "description": "the retention period for the metric at the specified timegrain.  Expressed as a duration 'PT1M', 'P1D', etc."
+        }
+      },
+      "description": "Metric availability specifies the time grain (aggregation interval or frequency) and the retention period for that time grain."
+    },
+    "AggregationType": {
+      "type": "string",
+      "description": "the aggregation type of the metric.",
+      "enum": [
+        "None",
+        "Average",
+        "Count",
+        "Minimum",
+        "Maximum",
+        "Total"
+      ],
+      "x-ms-enum": {
+        "name": "AggregationType",
+        "modelAsString": false
+      }
+    },
+    "MetricAggregationType": {
+      "type": "string",
+      "description": "the aggregation type of the metric.",
+      "enum": [
+        "None",
+        "Average",
+        "Count",
+        "Minimum",
+        "Maximum",
+        "Total"
+      ],
+      "x-ms-enum": {
+        "name": "MetricAggregationType",
+        "modelAsString": true
+      }
+    },
+    "MetricClass": {
+      "type": "string",
+      "description": "The class of the metric.",
+      "enum": [
+        "Availability",
+        "Transactions",
+        "Errors",
+        "Latency",
+        "Saturation"
+      ],
+      "x-ms-enum": {
+        "name": "MetricClass",
+        "modelAsString": true
+      }
+    },
+    "MetricDefinition": {
+      "type": "object",
+      "properties": {
+        "isDimensionRequired": {
+          "type": "boolean",
+          "description": "Flag to indicate whether the dimension is required."
+        },
+        "resourceId": {
+          "type": "string",
+          "description": "the resource identifier of the resource that emitted the metric."
+        },
+        "namespace": {
```

Comment:
> "namespace": { [](http://example.com/codeflow?start=7&length=15)

can this namespace be different from the namespace that the resourceId belongs to ?